.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
@hachreak Yes, I generally agree with the advantages... I was also concerned that the example instructions are kind of "duplicated" in `app.py` docstring and in `test_example_app.py`... Not very pretty.  However, having explicit instructions in "app.py" creates nicely readable documentation, as we did in e.g. [Flask-Menu](http://flask-menu.readthedocs.io/en/latest/#simple-example). Hence, if we do separate "setup" and "teardown" into shell scripts, we should think about propagating this nicely to the documentation.  (E.g. as we did for Invenio scripts[*].) This could come as part of the [better documentation structure](https://github.com/inveniosoftware/cookiecutter-invenio-module/issues/56). Let's muse IRL.

[*] BTW, there, we use naming technique "provision-something" , "install-something-else", "create-something", "populate-something" -- see https://github.com/inveniosoftware/invenio/tree/maint-1.2/scripts. So for the consistency reasons it would be better to stick to this naming order, "setup-example-app.sh", "populate-example-db.sh", "run-example-app.sh", "drop-example-db.sh", etc. Could muse about it IRL at the same time.